Cargo.toml

```diff
@@ -70,7 +70,7 @@ sched = ["process"]
 signal = ["process"]
 socket = []
 term = []
-time = []
+time = ["signal"]
```

Comment:
Rather than make all of time depend on signal, I suggest you just conditionalize the timer module on `#[cfg(all(feature = "time", feature = "signal"))]`.